src/binary-encoding.adoc

```diff
@@ -0,0 +1,1600 @@
++=========+
+----
+
+==== Opcode `0x71`: Month (2 bytes)
+[source]
+----
++=========+=========+
+|YYYY:YYYM|MMM-:----|
++=========+=========+
+----
+
+==== Opcode `0x72`: Day (2 bytes)
+[source]
+----
++=========+=========+
+|YYYY:YYYM|MMMD:DDDD|
++=========+=========+
+----
+
+==== Opcode `0x73`: Hour+Minutes @ UTC or Unknown (4 bytes)
+
+NOTE: Each encoding for a precision greater than or equal to `Hour+Minutes` comes in two flavors: one that uses a single
+bit (`U`) to indicate UTC versus Unknown offset, and another that uses 7 bits (`o`) to encode the number of quarter-hours
+offset from `-14:00`.
+
+[source]
+----
++=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmU:----|
++=========+=========+=========+=========+
+----
+
+==== Opcode `0x74`: Hour+Minutes @ Offset (5 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmo:oooo|oo--:----|
++=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x75`: Seconds @ UTC or Unknown (5 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmU:ssss|ss--:----|
++=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x76`: Seconds @ Offset (5 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmo:oooo|oo--:----|
++=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x77`: Milliseconds @ UTC or Unknown (6 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmk:ssss|ssff:ffff|ffff:----|
++=========+=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x78`: Milliseconds @ Offset (7 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmo:oooo|ooss:ssss|ffff:ffff|ff--:----|
++=========+=========+=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x79`: Microseconds @ UTC or Unknown (7 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmk:ssss|ssff:ffff|ffff:ffff|ffff:ff--|
++=========+=========+=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x7A`: Microseconds @ Offset (8 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmo:oooo|ooss:ssss|ffff:ffff|ffff:ffff|ffff:----|
++=========+=========+=========+=========+=========+=========+=========+=========+
+----
+
+==== Opcode `0x7B`: Nanoseconds @ UTC or Unknown (8 bytes)
+[source]
+----
++=========+=========+=========+=========+=========+=========+=========+=========+
+|YYYY:YYYM|MMMD:DDDD|HHHH:Hmmm|mmmo:oooo|ooss:ssss|ffff:ffff|ffff:ffff|ffff:----|
++=========+=========+=========+=========+=========+=========+=========+=========+
```

Comment:
I think this one needs `o`s replaced with `U` and more `f`s?

It's the same `f`s as microseconds, and same in the next entry, it's late and I can't math to figure out what's wrong.